_config.yml

```diff
@@ -79,6 +77,10 @@ collections:
     permalink: /:collection/:title
 
 defaults:
+  - scope:
+      path: ""
+    values:
+      image: /img/uswds-logo/lg-black.png
```

Comment:
It made more sense to move fallback image up since it's expected to be overridden.